src/misc/lv_log.c

```diff
@@ -111,6 +104,15 @@ void lv_log_add(lv_log_level_t level, const char * file, int line, const char *
                         lvl_prefix[level], LOG_TIMESTAMP_EXPR func, msg LOG_FILE_LINE_EXPR);
             custom_print_cb(level, buf);
         }
+        else {
+#if LV_LOG_PRINTF
```

Comment:
I think you can put the whole else block inside of the ifdef instead of making it noop.

Is there any special reason to make it noop?

Reply:
No particular reason. I have adjusted the code.